src/web/mjs/connectors/Cycomi.mjs

```diff
@@ -8,54 +8,123 @@ export default class CyComi extends Connector {
         super.id = 'cycomi';
         super.label = 'CyComi';
         this.tags = ['manga', 'japanese'];
-        this.url = 'https://cycomi.com';
+        this.url = 'https://web.cycomi.com';
     }
 
     async _getMangaFromURI(uri) {
         const request = new Request(uri, this.requestOptions);
-        const id = uri.pathname;
-        const title = (await this.fetchDOM(request, 'div.title-texts > h3'))[0].textContent.trim();
+        const id = uri.pathname.split("/").pop();
+        const title = (await this.fetchDOM(request, 'h2.css-m40re5'))[0].textContent.trim();
         return new Manga(this, id, title);
     }
 
     async _getMangas() {
+        const uri = new URL('/api/title/serialization/list', this.url);
```

Comment:
Will this endpoint list all manga or just serialised manga?

Reply:
I knew i'd have stuff to fix when i remembered you committed some Cycomi changes on Haru :D